rmf_fleet_adapter/include/rmf_fleet_adapter/agv/EasyFullControl.hpp

```diff
@@ -706,6 +712,12 @@ class EasyFullControl::FleetConfiguration
   /// Set whether or not to account for battery drain during task planning.
   void set_account_for_battery_drain(bool value);
 
+  /// Get whether or not to trigger automatic retreat to charger.
+  bool retreat_to_charger() const;
+
+  /// Set whether or not to trigger automatic retreat to charger.
+  void set_retreat_to_charger(bool value);
+
```

Comment:
I don't think these APIs should be added here. Instead they should be present in `FleetUpdateHandle` instead. Users who rely on `EasyFullControl` API can get a `FleetUpdateHandler` handler using the [EasyFullControl::more() API](https://github.com/open-rmf/rmf_ros2/blob/a2269abafa827195690e6e83d9b2a1b9d552d3f8/rmf_fleet_adapter/include/rmf_fleet_adapter/agv/EasyFullControl.hpp#L134-L135) and set this config.

Reply:
See comment below on setting/getting this value as an `std::optional<rmf_traffic::Duration>`.